addon/components/hyper-table/filters-renderers/date.js

```diff
@@ -1,96 +1,110 @@
+  @tracked flatpickrRef = null;
+  @tracked filterOption;
+  @tracked _currentMovingDateOption;
+  @tracked _currentDateValue;
+
+  constructor(owner, args) {
+    super(owner, args);
+
+    let filter = this.args.column.filters.find((f) => f.key === 'value');
+    this._currentMovingDateOption = filter ? filter.value : null;
+    this.filterOption = this._currentMovingDateOption ? 'moving' : 'fixed';
+
+    let lowerBoundFilter = this.args.column.filters.findBy('key', 'lower_bound');
+    let upperBoundFilter = this.args.column.filters.findBy('key', 'upper_bound');
+    if (lowerBoundFilter && upperBoundFilter) {
+      this._currentDateValue = [
+        moment.unix(lowerBoundFilter.value).toDate(),
+        moment.unix(upperBoundFilter.value).toDate()
+      ];
+    } else {
+      this._currentDateValue = [];
+    }
+  }
 
-export default Component.extend(FiltersRendererMixin, {
-  orderingOptions: computed('column.orderKey', function () {
+  get orderingOptions() {
     return {
-      'Oldest — Newest': `${this.column.orderKey}:asc`,
-      'Newest — Oldest': `${this.column.orderKey}:desc`
+      'Oldest — Newest': `${this.args.column.orderKey}:asc`,
+      'Newest — Oldest': `${this.args.column.orderKey}:desc`
     };
-  }),
+  }
 
-  filteringOptions: {
+  filteringOptions = Object.freeze({
     Moving: 'moving',
     Fixed: 'fixed'
-  },
-
-  filterOption: computed('currentMovingDateOption', function () {
-    if (!this.currentMovingDateOption) {
-      return 'fixed';
-    }
-
-    return 'moving';
-  }),
-
-  currentMovingDateOption: computed('column.filters', function () {
-    let filter = this.column.filters.find((f) => f.key === 'value');
-    return filter ? filter.value : null;
-  }),
-
-  _currentDateValue: computed('column.filters', function () {
-    let lowerBound = this.column.filters.findBy('key', 'lower_bound');
-    let upperBound = this.column.filters.findBy('key', 'upper_bound');
+  });
 
-    if (lowerBound && upperBound) {
-      return [moment.unix(lowerBound.value).toDate(), moment.unix(upperBound.value).toDate()];
-    }
-
-    return [];
-  }),
-
-  movingDateOptions: {
+  movingDateOptions = Object.freeze({
     Today: 'today',
     Yesterday: 'yesterday',
     'This Week': 'this_week',
     'Last Week': 'last_week',
     'This Month': 'this_month',
     'This Year': 'this_year'
-  },
-
-  actions: {
-    orderingOptionChanged(value) {
-      this.manager.updateOrdering(this.column, value);
-    },
-
-    filterOptionChanged(value) {
-      this.set('filterOption', value);
-    },
-
-    selectMovingDate(value) {
-      this.set('column.filters', [
-        {
-          key: 'value',
-          value: value
-        }
+  });
+
+  _handleFlatpickrClick(e) {
+    e.stopPropagation();
+  }
+
+  @action
+  openedFlatpickr(...args) {
+    this.calendarContainer = args[2].calendarContainer;
+    this.calendarContainer.addEventListener('click', this._handleFlatpickrClick);
+  }
```

Comment:
@Miexil the pr escalated a bit but here is the fix for what you said. It now targets more specifically the calendar triggered by the component.